{"name": "Попытка авторизации с неверным паролем", "status": "passed", "start": 1748556412731, "stop": 1748556412916, "uuid": "a0c7a22c-dc65-481a-8836-f1ec8f93a013", "historyId": "9a20e2fed9419e29a8a33c83205fd1e0", "testCaseId": "9a20e2fed9419e29a8a33c83205fd1e0", "fullName": "tests.couriers.test_courier_login.TestCourierLogin#test_authorization_with_invalid_password", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "EgorLaptop"}, {"name": "thread", "value": "29732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_courier_login"}]}